src/components/PieChart/PieChart.jsx

```diff
@@ -30,6 +30,7 @@ const styles = {
   }
 }
 
+// TODO: should be in cozy-ui
```

Comment:
I don't see it in the figma : https://www.figma.com/file/7e3nYZkJUGZ7SLqy8xWbqx/Cozy-UI

Reply:
y'en a pas en effet ping @joel-costa